Angular/src/app/components/filter-form/config.ts

```diff
@@ -0,0 +1,15 @@
+export interface IFilterInputOptions {
+  placeholder: string
+  onKeyUp: any
+}
+
+export interface IDateOptions {
+  value: Date
+  onValueChanged: any
```

Comment:
Try to avoid 'any' type, because it's a little dangerous and breaks the type's system.

In this case, you can use the next type:
```ts
onValueChanged: (event: OptionChangedEvent) => void;
```